src/renderer/viz/expressions/aggregation/viewport/ViewportHistogram.js

```diff
@@ -75,10 +74,102 @@ export default class ViewportHistogram extends BaseExpression {
+     * const numberOfWheels = [
+     *  { key: 'car', value: 4 },
+     *  { key: 'truck', value: 8 },
+     *  { key: 'bike', value: 2 }
+     * ];
+     *
+     * const s = carto.expressions;
+     * const viz = new carto.Viz({
+     *   @histogram: s.viewportHistogram(s.prop('vehicles))
+     * });
+     *
+     * const data = viz.variables.histogram.getJoinedValues(numberOfWheels);
+     * // returns an array with the following format:
+     * // [
+     * //   { frequency: 10, key: 'truck', value: 8 }
+     * //   { frequency: 20, key: 'bike', value: 2 }
+     * //   { frequency: 30, key: 'car', value: 4 }
+     * // ]
+     *
+     * @example <caption>Get joined data for a categorical property sorted by frequency. (String)</caption>
+     * const numberOfWheels = [
+     *  { key: 'car', value: 4 },
+     *  { key: 'truck', value: 8 },
+     *  { key: 'bike', value: 2 }
+     * ];
+     *
+     * const s = carto.expressions;
+     * const viz = new carto.Viz(`
+     *   @histogram: viewportHistogram($vehicles)
+     * `);
+     *
+     * const data = viz.variables.histogram.getJoinedValues(numberOfWheels);
+     * // returns an array with the following format:
+     * // [
+     * //   { frequency: 10, key: 'truck', value: 8 }
+     * //   { frequency: 20, key: 'bike', value: 2 }
+     * //   { frequency: 30, key: 'car', value: 4 }
+     * // ]
+     *
+     * @example <caption>Get color values for the histogram when using a ramp.</caption>
+     *
+     * const s = carto.expressions;
+     * const viz = new carto.Viz(`
+     *   @histogram: s.viewportHistogram(s.prop('vehicles'))
+     *   color: ramp(s.prop('vehicles'), s.palettes.PRISM)
+     * `);
+     *
+     * const legendData = viz.color.getLegendData();
+     * const data = viz.variables.histogram.getJoinedValues(legendData);
+     * // returns the following array
+     * // [
+     * //   { frequency: 10, key: 'truck', value: { r: 95, g: 70, b: 144, a: 1 } }
+     * //   { frequency: 20, key: 'bike', value: { r: 29, g: 105, b: 150, a: 1 } }
+     * //   { frequency: 30, key: 'car', value: { r: 56, g: 166, b: 165, a: 1 } }
+     * // ]
+     *
+     * @example <caption>Get color values for the histogram when using a ramp. (String)</caption>
+     *
+     * const s = carto.expressions;
+     * const viz = new carto.Viz(`
+     *   @histogram: viewportHistogram($vehicles)
+     *   color: ramp($vehicles, Prism)
+     * `);
+     *
+     * const legendData = viz.color.getLegendData();
+     * const data = viz.variables.histogram.getJoinedValues(legendData);
+     * // returns the following array
+     * // [
+     * //   { frequency: 10, key: 'truck', value: { r: 95, g: 70, b: 144, a: 1 } }
+     * //   { frequency: 20, key: 'bike', value: { r: 29, g: 105, b: 150, a: 1 } }
+     * //   { frequency: 30, key: 'car', value: { r: 56, g: 166, b: 165, a: 1 } }
+     * // ]
+     *
+    */
+    getJoinedValues (values) {
```

Comment:
✨ Nice useful method!

Reply:
Just a question, doesn't values come from histogram itself? In that case we could remove the parameter. Regarding to its name, maybe it could be clearer without the "joined" word... how about `getFrequencyValues`?

Reply:
But this is actually joining the results of the histogram with the ones of the legend

Reply:
I see... It's just that the name didn't sound very clear to me

Reply:
After asking people, we're going to keep using `getJoinedValues`, but in the documentation it is explained that it is sorted by frequency.